packages/features/users/components/UserTable/UserListTable.tsx

```diff
@@ -400,6 +401,18 @@ export function UserListTable() {
 
   const numberOfSelectedRows = table.getSelectedRowModel().rows.length;
 
+  const handleDownload = () => {
```

Comment:
The issue here is this is all client side. So deel would need to load all data then do the export. Not sure if this is blocking

Reply:
Good point but handling this server-side deserves a series of big refactors + we do these things client side already in e.g., insights, routing form reports